### PR TITLE
fix issue https://github.com/plone/plone.app.widgets/issues/111

### PR DESCRIPTION
**Pull request by yboussard**
correction for https://github.com/plone/plone.app.widgets/issues/111

### CODE REVIEW

**Comment by vangheem** [comment]
@thet @frapell want to look over this? I don't use 1.x of plone.app.widgets.

**Comment by vangheem** [comment]
Also, if this is necessary, we'll want to this backported to Products.Archetypes.

**Comment by jensens** [comment]
@yboussard youre not shown as plone contributor. please sign the contributor agreement, otherwise we cant merge youre PR because of legal reasons. for information read https://plone.org/foundation/contributors-agreement

**Comment by jensens** [comment]
changelog entry is missing

File: plone/app/widgets/base.py [hunk 1]
```diff
@@ -215,6 +221,7 @@ def _get_value(self):
         :returns: Returns list of selected option(s) values.
         :rtype: list
         """
+        import pdb;pdb.set_trace();
```
Comment (petschki) [inline plone/app/widgets/base.py]: @yboussard erm ....

**Comment by jensens** [comment]
Hence there was no activity for a long time, I propose the to close this PR within next two weeks. If you do not feel ok with this, please speak up and provide us a rough schedule.

**Comment by jensens** [comment]
close this in favour of #149

**Pull request closed without merging**